packages/react-variant-view/src/component/frequencyTable/FrequencyTable.css

```diff
@@ -0,0 +1,18 @@
+/* this component comes from signal */
```

Comment:
Should we remove this comment? After merging this PR, we are going to remove the one in Signal repo and import this one, right?